doc/source/conf.py

```diff
@@ -589,6 +590,25 @@ def get_version_match(semver):
 copybutton_prompt_text = r'>>> ?|\.\.\. '
 copybutton_prompt_is_regexp = True
 
+# sphinx-tags options ---------------------------------------------------------
+# See https://sphinx-tags.readthedocs.io/en/latest/index.html
+
+tags_badge_colors = {
+    "Load": "primary",
+    "Filter": "secondary",
+    "Plot": "dark",
+    "Widgets": "success",
+    "Lights": "primary",
+}
+tags_create_tags = True
+tags_create_badges = True
+tags_index_head = "Gallery examples categorised by tag:"  # tags landing page intro text
+tags_intro_text = "🏷 Tags:"  # prefix text for a tags list
+tags_overview_title = "🏷 Tags"  # title for the tags landing page
+tags_output_dir = "tags"
+tags_page_header = "Gallery examples contain this tag:"  # tag sub-page, header text
+tags_page_title = "🏷 Tag"  # tag sub-page, title appended with the tag name
```

Comment:
I know this is my second anti-emoji comment today, sorry :smile: But can we please remove the emoji?
![image](https://github.com/user-attachments/assets/f36755d2-0688-4cfa-9ea7-eaa0c0d2e353)
```suggestion
tags_intro_text = "Tags:"  # prefix text for a tags list
tags_overview_title = "Tags"  # title for the tags landing page
tags_output_dir = "tags"
tags_page_header = "Gallery examples contain this tag:"  # tag sub-page, header text
tags_page_title = "Tag"  # tag sub-page, title appended with the tag name
```

Reply:
You're a good emoji hunter.